scene/resources/mesh.cpp

```diff
@@ -902,7 +902,7 @@ void Mesh::clear_cache() const {
 
 #ifndef _3D_DISABLED
 Vector<Ref<Shape3D>> Mesh::convex_decompose(const Ref<MeshConvexDecompositionSettings> &p_settings) const {
-	ERR_FAIL_NULL_V(convex_decomposition_function, Vector<Ref<Shape3D>>());
+	ERR_FAIL_NULL_V_MSG(convex_decomposition_function, Vector<Ref<Shape3D>>(), "No convex decomposition function available. It should be provided by the V-HACD module.");
```

Comment:
Use the same error message as [here](https://github.com/godotengine/godot/blob/594d64ec244b6b99321e2096987d4d69de4c8845/core/io/image.cpp#L4092):

```suggestion
	ERR_FAIL_NULL_V_MSG(convex_decomposition_function, Vector<Ref<Shape3D>>(), "The V-HACD module isn't enabled. Recompile the Godot editor or export template binary with the `module_vhacd_enabled=yes` SCons option.");
```